govcd/vapp_test.go

```diff
@@ -1441,3 +1441,122 @@ func (vcd *TestVCD) Test_AddNewVMFromMultiVmTemplate(check *C) {
 	check.Assert(err, IsNil)
 	check.Assert(task.Task.Status, Equals, "success")
 }
+
+// Test_AddNewVMWithComputeCapacity creates a new VM in vApp with VM using compute capacity
+func (vcd *TestVCD) Test_AddNewVMWitComputeCapacity(check *C) {
+
+	if vcd.client.Client.APIVCDMaxVersionIs("< 33.0") {
+		check.Skip(fmt.Sprintf("Test %s requires vCD 10.0 (API version 33) or higher", check.TestName()))
```

Comment:
We should use the new abbreviation capitalisation `VCD` in all the new code.
```suggestion
		check.Skip(fmt.Sprintf("Test %s requires VCD 10.0 (API version 33) or higher", check.TestName()))
```

Reply:
fixed